notary-registration/src/main/kotlin/com/d3/commons/registration/NotaryRegistrationConfig.kt

```diff
@@ -26,8 +26,11 @@ interface NotaryRegistrationConfig {
     /** Iroha account to store clients */
     val clientStorageAccount: String
 
+    /** Property indicating if a brvs used in the system **/
+    val isBrvsEnabled: Boolean
+
     /** Iroha account of validator service */
-    val brvsAccount: String
+    val brvsAccount: String?
```

Comment:
Do we really need it nullable?

Reply:
Otherwise there is an exception thrown if there is no 'brvsAccount' property specified which is quite honest if 'brvsEnabled' is set to `false`